packages/edit-site/src/components/sidebar-edit-mode/index.js

```diff
@@ -69,7 +69,7 @@ export function SidebarComplementaryAreaFills() {
 				identifier={ sidebarName }
 				title={ __( 'Settings' ) }
 				icon={ isRTL() ? drawerLeft : drawerRight }
-				closeLabel={ __( 'Close settings sidebar' ) }
```

Comment:
It's called `Close settings` everywhere else, so it made sense to me to make it consistent.